README.md

```diff
@@ -210,3 +211,23 @@ We are deliberately not using other project dependencies (notably `the-build`, `
 Solr support is not provided by default. It can be added later if needed.
 
 The `main` branch is locked against commits that are not in approved pull requests.
+
+
+## Services
+
+### Solr server
+
+A solr server is ready to be used at http://drupal-10-development.ddev.site:8983/solr/.
+
+`search_api_solr` server dependency is not currently installed, if you want to test the functionality try this branch [search_api_example](https://github.com/palantirnet/drupal-10-development/tree/search_api_example) or follow these steps.
+  * Install `search_api_solr` module `composer require drupal/search_api_solr`
+  * Enable the `search_api_solr` module either using the web interface or `ddev drush en -y search_api_solr`
+  * Navigate to [Search api page](https://drupal-10-development.ddev.site/admin/config/search/search-api), click "Add server button" 
+  * Choose Solr as backend.
+  * Choose "Standard" as the Solr connector type and configure it:
+    * Set "Server name" to anything you want. Maybe "ddev-solr-server".
+    * Set "Backend" to "Solr"
+    * Set "Solr host" to solr.
+    * Set "Solr core" name to "dev".
+    *Under "Advanced server configuration" set the "solr.install.dir" to /opt/solr.
+  * `ddev restart`
```

Comment:
I would list the `ddev restart` prior to the module configuration, since we expect the configuration to "work" when completed.